handbook/openid-connect/openid-connect.md

```diff
@@ -316,3 +316,18 @@ The following sources can be used to troubleshoot a failed connection.
     SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.
 
 **Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).
+
+### Cache Shm Entry Size Max
+
+If an "Internal Server Error" is encountered when attempting to log in, this may be an issue with the defined `OIDCCacheShmEntrySizeMax`. You can verify this is the issue by searching the NI Web Server logs for the following entries
+```
+    oidc_cache_shm_set: could not store value since value size is too large
+    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
+```
+To resolve this issue
+
+- Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf.defaults`
```

Comment:
50_mod_auth_openidc-defines.conf.defaults is used to create 50_mod_auth_openidc-defines.conf on install, and to merge in new settings on upgrade. The server uses the configuration in 50_mod_auth_openidc-defines.conf.

```suggestion
- Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf`
```